src/quo2/components/token_tag.cljs

```diff
@@ -0,0 +1,68 @@
+(ns quo2.components.token-tag
+  (:require [quo2.foundations.colors :as colors]
+            [quo.react-native :as rn]
+            [quo.theme :as theme]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.components.text :as text]))
+
+(def themes {:light   {:background-color colors/neutral-20}
```

Comment:
Indentation is a bit off, we either leave a single space, or align values vertically